src/tabs/tab-headers.component.ts

```diff
@@ -135,7 +135,7 @@ export class TabHeaders implements AfterViewInit, AfterContentInit {
 	 */
 	@HostListener("keydown", ["$event"])
 	keyboardInput(event) {
-		if (event.key === "ArrowRight" || event.key === "ArrowDown") {
+		if (event.key === "Right" || event.key === "ArrowRight") {
```

Comment:
I was under the impression that ArrowUp/ArrowDown were also valid ... Also which browser sends just Right? That may need a comment

Reply:
Ah Interesting ... I'd still add a comment for the IE11-ness of "Right", since it's nonstandard